app/javascript/gobierto_dashboards/webapp/containers/costs/Home.vue

```diff
@@ -0,0 +1,138 @@
+        <div class="gobierto-dashboards-container-title">
+          <h2 class="pure-u-1 gobierto-dashboards-title gobierto-dashboards-title-select">
+            {{ labelTittle }}
+          </h2>
+          <select
+            v-model="yearFiltered"
+            class="form-control gobierto-dashboards-select"
+            @change="onChangeFilterYear"
+          >
+            <option
+              v-for="year in years"
+              :key="year"
+              :value="year"
+              :index="year"
+              class="gobierto-dashboards-select-option"
+            >
+              {{ year }}
+            </option>
+          </select>
+        </div>
+        <p class="gobierto-dashboards-description">
+          {{ labelDescription }}
+        </p>
+        <p class="gobierto-dashboards-description">
+          {{ labelDescription2 }}
+        </p>
+        <p class="gobierto-dashboards-description">
+          {{ labelDescription3 }}
+        </p>
+      </div>
+      <Distribution
+        :data="groupData"
+        :year="yearFiltered"
+        :years="years"
+        @preventReload="injectRouter"
+      />
+      <Table
+        :items-filter="groupDataFilter"
+        :items="costDataFilter"
+        :year="yearFiltered"
+        :base-title="baseTitle"
+      />
+    </div>
+  </main>
+</template>
+<script>
+import Distribution from './Distribution.vue'
+import Table from './table/Table.vue'
+export default {
+  name: 'Home',
+  components: {
+    Distribution,
+    Table
+  },
+  data() {
+    return {
+      costData: this.$root.$data.costData,
+      groupData: this.$root.$data.groupData,
+      getSiteName: document.querySelector('[data-site-name]').getAttribute('data-site-name'),
+      labelTittle: I18n.t("gobierto_dashboards.dashboards.costs.title") || "",
+      labelDescription: '',
+      labelDescription2: I18n.t("gobierto_dashboards.dashboards.costs.description_2") || "",
+      labelDescription3: I18n.t("gobierto_dashboards.dashboards.costs.description_3") || "",
+      yearFiltered: "2018",
+      years: ['2018', '2019'],
+      costDataFilter: [],
+      groupDataFilter: []
+    }
+  },
+  watch: {
+    $route(to) {
+      if (to.name) {
+        this.injectRouter()
+      }
+    }
+  },
+  created() {
+      this.labelDescription = I18n.t("gobierto_dashboards.dashboards.costs.description", { entity_name: this.getSiteName }) || "";
+    const {
+      params: {
+        year: year
+      }
+    } = this.$route
+    let yearFiltered = year
+    if (!year) yearFiltered = '2018'
+    this.yearFiltered = yearFiltered
+
+    this.costDataFilter = this.costData.filter(element => element.year === yearFiltered)
+    this.groupDataFilter = this.groupData.filter(element => element.year === yearFiltered)
+
+    this.costDataFilter = this.costDataFilter.sort((a, b) => (a.cost_total > b.cost_total) ? -1 : 1)
+    this.groupDataFilter = this.groupDataFilter.sort((a, b) => (a.cost_total > b.cost_total) ? -1 : 1)
```

Comment:
When you do multiple operations over the same element, it's better to use local variables to avoid render cycles once assigned.
In this case, you could do it in the same line, since is straight-forward. But, you should get use to update the reactive prop once "the job is done"